_static/files/audit-event-examples/R4_read.json

```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://hl7.org/fhir/audit-event-type",
+        "code": "rest",
+        "display": "RESTful Operation"
+    },
+    "subtype": [
+        {
+            "system": "http://hl7.org/fhir/http-verb",
+            "code": "GET"
+        },
+        {
+            "system": "http://hl7.org/fhir/restful-interaction",
+            "code": "read"
+        }
+    ],
+    "action": "R",
+    "recorded": "2022-10-07T14:21:15.456+00:00",
+    "outcome": "0",
+    "outcomeDesc": "200",
+    "agent": [
+        {
+            "altId": "unknown",
+            "who": {
+                "display": "anonymous"
+            },
+            "type": {
+                "coding": [
+                    {
+                        "system": "https://profiles.ihe.net/ITI/BALP/CodeSystem/UserAgentTypes",
+                        "code": "UserOauthAgent",
+                        "display": "User OAuth Agent participant"
+                    }
+                ]
+            },
+            "requestor": true,
+            "network": {
+                "address": "127.0.0.1",
+                "type": "2"
+            }
+        },
+        {
+            "type": {
```

Comment:
An empty agent with no content is not useful. It's the client right? I would merge it with the other agent or move the network property down here. Also add another "requestor": "true" to it.

Reply:
I want to discuss it with you in a call to make sure I understand you